README.md

```diff
@@ -1,2 +1,494 @@
+
+
+### Run the entire test suite in parallel on multiple BrowserStack browsers
+
+In this section, we will run the tests in parallel on multiple browsers on Browserstack. Refer to the `parallel` object in `bs.json` file to change test capabilities for this configuration.
+
+- How to run the test?
+
+  To run the entire test suite in parallel on multiple BrowserStack browsers, use the following command:
+
+  Maven:
+  ```sh
+  mvn test -P bstack-parallel-browsers
+  ```
+
+  Gradle:
+  ```sh
+  gradle bstack-parallel-browsers
+  ```
+
+### [Web application hosted on internal environment] Running your tests on BrowserStack using BrowserStackLocal
+
+#### Prerequisites
+
+- Clone the [BrowserStack demo application](https://github.com/browserstack/browserstack-demo-app) repository.
+  ```sh
+  git clone https://github.com/browserstack/browserstack-demo-app
+  ``` 
+- Please follow the README.md on the BrowserStack demo application repository to install and start the dev server on localhost.
+- In this section, we will run a single test case to test the BrowserStack Demo app hosted on your local machine i.e. localhost. Refer to the `single_local` object in `bs.json` file to change test capabilities for this configuration.
+- Note: You may need to provide additional BrowserStackLocal arguments to successfully connect your localhost environment with BrowserStack infrastructure. (e.g if you are behind firewalls, proxy or VPN).
+- Further details for successfully creating a BrowserStackLocal connection can be found here:
+
+  - [Local Testing with Automate](https://www.browserstack.com/local-testing/automate)
+  - [BrowserStackLocal Java GitHub](https://github.com/browserstack/browserstack-local-java)
+
+
+### [Web application hosted on internal environment] Run a specific test on BrowserStack using BrowserStackLocal
+
+- How to run the test?
+
+  - To run the default test scenario (e.g. End to End Scenario) on a single BrowserStack browser using BrowserStackLocal, use the following command:
+
+  Maven:
+  ```sh
+  mvn test -P bstack-local
+  ```
+
+  Gradle:
+    ```sh
+  gradle bstack-local
+  ```
+
+  To run a specific test scenario, use the following command with the additional test-name argument:
+  Maven:
+  ```sh
+  mvn test -P bstack-local -Dtest-name="<Test scenario name>"
+  ```
+
+  Gradle:
+  ```sh
+  gradle bstack-local -Dtest-name="<Test scenario name>"
+  ```
+
+  where,  the argument 'test-name' can be any Cucumber scenario name configured in this repository.
+
+  E.g. "Login as username", "Login as Locked User", "Offers for mumbai geo-location" or any of the other test scenario names, as outlined in [About the tests in this repository](#About-the-tests-in-this-repository) section.
```

Comment:
`mvn test -P on-prem -Dtest-name="Offers for mumbai geo-location"`
`mvn test -P on-prem -Dtest-name="Login as username"`

Both these profiles do not trigger any test.

Reply:
The profile names are different than the one's mentioned in the README.md
This is the right command:
 `mvn test -P on-prem -Dtest-name="Login with given '<username>' user" `
` mvn test -P on-prem -Dtest-name="Offers for Mumbai location"`        
We will make the changes.